pkg/apis/apis.go

```diff
@@ -11,7 +13,16 @@ var AddToSchemes runtime.SchemeBuilder
 
 // AddToScheme adds all Resources to the Scheme
 func AddToScheme(s *runtime.Scheme) error {
-	return AddToSchemes.AddToScheme(s)
+	err := AddToSchemes.AddToScheme(s)
+	if err != nil {
+		return errors.New("AddToSchemes Not added to scheme")
+	}
+	err = routev1.AddToScheme(s)
```

Comment:
where are the other AddToSChemes for the other apis we use ?

Reply:
They have been registered in `register.go`